### PR TITLE
0.47b Build

### PR DESCRIPTION
**Pull request by Arishamays1**
## ☆ Opentelemetry-instrumentation 0.47b0 ☆
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-6098)
[Upstream](https://github.com/open-telemetry/opentelemetry-python/tree/main)

### Changes
 - Updated version number and sha256
- Updated `about` section
- Updated deps

### CODE REVIEW

**Review by cbouss — APPROVED** [review]

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+aggregate_check: False
```
Comment (lorepirri) [inline abs.yaml:1]: why this?

**Comment by lorepirri** [comment]
https://github.com/open-telemetry/opentelemetry-python-contrib/tree/v0.47b0/opentelemetry-instrumentation

**Review by lorepirri — APPROVED** [review]

File: abs.yaml (hunk 1 above)
Comment (Arishamays1) [reply abs.yaml:1]: This is a PR for opentel-instrum, targeting a version lower than the latest available, which I recently updated. Without it, the builds fail. This ticket is to rebuild all the deps and packages within opentelemetry-python 1.26

Comment (lorepirri) [reply abs.yaml:1]: ok thanks for the details!

**Pull request merged**